pkg/resources/cruisecontrol/cruisecontrol.go

```diff
@@ -125,3 +150,15 @@ func (r *Reconciler) getClientPassword() (string, error) {
 
 	return clientPass, nil
 }
+
+func isBrokerDeletionInProgress(brokers []v1beta1.Broker, brokerState map[string]v1beta1.BrokerState) bool {
+	if len(brokers) < len(brokerState) {
```

Comment:
Do we need this check here or the `for` loop below should be enough?

Could it happen during upscale that the number of brokers is different from the number of broker states?